src/CreeDictionary/phrase_translate/management/commands/translatewordforms.py

```diff
@@ -157,10 +175,14 @@ def generate_translations(self) -> Iterable[tuple[Wordform, list[Definition]]]:
                     input_text = remove_parentheticals(definition.text)
 
                     phrase = inflect_english_phrase(wordform.analysis, input_text)
-                except UnknownTagError:
-                    raise Exception(
-                        f"Unknown tag for {wordform.text} {wordform.analysis}"
+                except UnknownTagError as e:
+                    logger.debug(
+                        f"Unknown tag in {wordform.text} {wordform.analysis}: {e}"
                     )
+                    self.translation_stats.unknown_tags_during_auto_translation[
+                        e.args[0]
+                    ] += 1
```

Comment:
Are the translation stats output somewhere?

Reply:
Yeah, the stats object is printed at the end of the `importjsondict` command.